### ISSUE TITLE
CiviCRM 4.7

### ISSUE THREAD

**Issue by osarusan82** [osseed/com.osseed.eventcalendar#21]
After upgrading to CiviCRM 4.7, the calendar no longer displays. Will this be updated to be compatible with 4.7?

**Comment by osarusan82** [comment]
I tried re-installing the extension, but it still doesn't work. CiviCRM is giving me the following error: Cannot set unrecognized property CRM_Core_Config::$civicrm_events_event_types

**Comment by audio-1** [comment]
we're having the same problem - are there any plans to make this compatible with 4.7?

**Comment by cdhassell** [comment]
It works for me on 4.7.3.  Are you sure that your extension directory did not change?  In 4.7 it seems to default to files/ext, which is a change from where I used to have it at files/extensions.

**Comment by audio-1** [comment]
Hi there -
I've tried making the changes as outlined in the patch, but its still not working. Any ideas?

Thanks!

**Comment by lizmestres** [comment]
Hi,
I have the same problem except that the calendar displays, but without the color coding. When I try to go to the event calendar settings, I get the message (as above)  "Sorry but we are not able to provide this at the moment. Cannot set unrecognized property CRM_Core_Config::$civicrm_events_event_types." I have applied the patch but still get the same result. Any help will be greatly appreciated.

**Comment by sushantpaste** [comment]
@lizmestres We have added fixes on https://github.com/osseed/com.osseed.eventcalendar/tree/develop branch you can give a try with Develop branch.

@cdhassell Thanks for your help.

**Comment by lizmestres** [comment]
Hi,
Thanks but I applied the fixes you posted but still have the same result--no color coding and the message at event calendar settings -- ""Sorry but we are not able to provide this at the moment. Cannot set unrecognized property CRM_Core_Config::$civicrm_events_event_types." What I did to apply fixes was delete the lines marked "-" and add the lines marked "+" in the specified files -- EventSettings.php and ShowEvents.php. However, I'm not a techie, just a designer, so is there something else I should do?
